content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,245 @@
+a :guilabel:`Team`. This will reveal that team's :guilabel:`Pipeline`. Click :guilabel:`New` to
+create a new ticket.
+
+On the blank ticket form, create a ticket :guilabel:`Title`, and enter the :guilabel:`Customer`
+information.
+
+When the customer name is added, the :guilabel:`Sales Order Item` field will automatically populate
+with the most recent prepaid sales order item that has time remaining.
+
+Track hours on helpdesk ticket
+------------------------------
+
+Time spent working on a *Helpdesk* ticket is tracked on the *Timesheets* tab on the specific ticket.
+
+On the ticket detail form, click on the :guilabel:`Timesheets` tab and click :guilabel:`Add a line`.
+Choose an :guilabel:`Employee`, add a :guilabel:`Description` of the task, and enter the number of
+:guilabel:`Hours Spent`.
+
+As new lines are added to :guilabel:`Timesheets` tab, the :guilabel:`Remaining Hours on SO` field is
+automatically updated.
+
+.. image:: track_and_bill/track-bill-remaining-hours-total.png
+   :align: center
+   :alt: View of the timesheets tab on a ticket with an emphasis on the remaining hours on an SO.
+
+.. note::
+   If the number of hours on the :guilabel:`Timesheets` tab exceeds the number of hours sold, the
+   :guilabel:`Remaining Hours of SO` will turn red.
+
+As hours are added to the :guilabel:`Timesheets` tab, they are automatically updated in the
+:guilabel:`Delivered` field on the :abbr:`SO (sales order)`, as well.
+
+Invoice post-paid support services
+==================================
+
+When support services are billed based on the amount of time spent on an issue, an invoice cannot be
+created before the total number of hours required to solve the problem have been entered on a
+timesheet. In this case, a service product with the invoicing policy set to :guilabel:`Based on
+Timesheets` (like the one created above) would be used.
+
+Create a sales order with a time-tracked product
+------------------------------------------------
+
+To invoice a customer for post-paid support services, first create a sales order (SO) with the
+*support services product*. To do this, go to :menuselection:`Sales --> Orders --> Quotations -->
+New`.
+
+Fill out the quotation with the customer information.
+
+On the :guilabel:`Order Lines` tab, click :guilabel:`Add a Product`. Select the post-paid services
+product configured in the steps above. After updating any other necessary information,
+:guilabel:`Confirm` the quotation.
+
+.. note::
+   Unlike with the prepaid services quotation, Odoo will not allow an invoice to be created at this
+   time. That is because no services have been performed. In other words nothing has been delivered.
+   Therefore, there is nothing to invoice.
+
+Create a helpdesk ticket for time-tracked services
+--------------------------------------------------
+
+To record a *Timesheet* entry for time-tracker services, go to :menuselection:`Helpdesk` and select
+the appropriate :guilabel:`Team`.
```

Comment:
no gui
```suggestion
the appropriate team for which these services apply.
```

Reply:
Same as I mentioned before, this is part of an action so I do not see why there would not be a guilabel for it?

Reply:
Because this is not the text as it appears on the UI.
See review comment above.